ray-on-gke/tpu/kuberay-tpu-webhook/webhook_main_test.go

```diff
@@ -1163,7 +988,7 @@ func Test_ValidateRayCluster(t *testing.T) {
 		},
 		"validateRayCluster worker group spec not compatible with gke-tpu-topology": {
 			// request TPUs, workers don't match topology, return false
-			rayCluster:      testRayCluster.DeepCopy(),
+			rayCluster:      getTestRayCluster("test-cluster", "test-group", "test-namespace", int32(2), 1),
 			topology:        "2x2x2",
```

Comment:
The `toplogy` and `numOfHost` entries in the test table are no longer needed since it's included in `getTestRayCluster`

Reply:
Done in [54b8a71](https://github.com/GoogleCloudPlatform/ai-on-gke/pull/723/commits/54b8a71d5298ec19cca58a5cc912e56f78e7e0fa), we still need to pass the gke-tpu nodeSelectors but I moved that to the `getTestRayCluster` function call.